### DISCUSSION TITLE
how to set timezone?

### DISCUSSION THREAD

**Discussion by Pojay-hcz** [gtxaspec/wz_mini_hacks#773] [Q&A] [answered]
I have 2xWyze Cam V3, installed with fw 4.36.9.139 and wz_mini_hacks built 2024-05-27. Both are connected to Frigate in Home Assistant. Since I block Internet access for all my cameras in our router I installed Chrony as addon in Home Assistant to relais the local NTP-pool into my LAN. My TZ is UTC+7. All Dahua cameras and a Reolink doorbell sync their local time perfectly with Chrony now. But 2 Wyze V3 show UTC-7 instead. 
I changed the timezone setting in /configs/.user_config with nano via SSH from -7 to 7 and rebooted the camera without success.
If anyone of you could guide me further I would appreciate a lot.

**Comment by virmaior** [comment, marked as answer]
the version of busybox running on wyze cameras doesn't actually understand time zones.

iCamera (the application that wyze uses to make the video files) uses `/configs/TZ`
the `/configs/.user_config`  value might be used in some other context...

**Reply by Pojay-hcz** [reply]
thank you for your answer
